### PR TITLE
[CHG] partner_contact_in_several_companies

### PR DESCRIPTION
**Pull request by valentinthirion**
Improved view for better clarity of the fonctions for a partner

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Code OK

File: partner_contact_in_several_companies/views/res_partner.xml [hunk 1]
```diff
@@ -114,6 +118,12 @@
                                     <field name="mobile" widget="phone"/>
                                     <field name="comment" placeholder="internal note..."/>
                                 </group>
+                                <div>
+                                    <group col="2">
+                                        <field name="image" widget="image" options="{'preview_image': 'image_medium', 'size': [90, 90]}" modifiers="{}"/>
+                                        <field name="category_id" widget="many2many_tags" placeholder="Étiquettes..." can_create="true" can_write="true" modifiers="{}"/>
```
Comment (MiquelRForgeFlow) [inline partner_contact_in_several_companies/views/res_partner.xml]: shouldn't be `tags` or `labels` instead of `Étiquettes`?

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by ghost** [comment]
:+1: LGTM

File: partner_related_documents/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (nikul-serpentcs) [inline partner_related_documents/__init__.py:1]: @valentinthirion Add New Blank Line

File: partner_related_documents/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    'name': "Partner Related Document",
+    'version': '9.0.1.0',
+    'depends': [
+        "website_sign",
+    ],
+    'author': "François Wyaime, AbAKUS it-solutions SARL",
+    'website': "http://www.abakusitsolutions.eu",
```
Comment (nikul-serpentcs) [inline partner_related_documents/__openerp__.py:8]: URL Should be `https://github.com/OCA/partner-contact`

File: partner_related_documents/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    'name': "Partner Related Document",
+    'version': '9.0.1.0',
+    'depends': [
+        "website_sign",
+    ],
+    'author': "François Wyaime, AbAKUS it-solutions SARL",
```
Comment (nikul-serpentcs) [inline partner_related_documents/__openerp__.py:7]: Added OCA in author list

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by NL66278** [comment]
@valentinthirion Will you look into solving the runbot and travis failures?

**Comment by NL66278** [comment]
@valentinthirion Please fix at least these:
```
partner_related_documents/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

************* Module partner_related_documents.__openerp__

partner_related_documents/__openerp__.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

partner_related_documents/__openerp__.py:1: [C8101(manifest-required-author), ] One of the following authors must be present in manifest: 'Odoo Community Association (OCA)'

partner_related_documents/__openerp__.py:1: [C8102(manifest-required-key), ] Missing required key "license" in manifest file

partner_related_documents/__openerp__.py:1: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file

partner_related_documents/__openerp__.py:1: [C8106(manifest-version-format), ] Wrong Version Format "9.0.1.0" in manifest file. Regex to match: "(9\.0)\.\d+\.\d+\.\d+$"

************* Module partner_contact_in_several_companies

partner_contact_in_several_companies/views/res_partner.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

partner_contact_in_several_companies/demo/ir_actions.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

partner_contact_in_several_companies/demo/res_partner.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

************* Module partner_contact_in_several_companies.models.res_partner

partner_contact_in_several_companies/models/res_partner.py:15: [C8108(method-compute), ResPartner] Name of compute method should start with "_compute_"

partner_contact_in_several_companies/models/res_partner.py:28: [W8104(api-one-deprecated), ResPartner._get_contact_type] api.one deprecated

************* Module partner_contact_in_several_companies.models.ir_actions

partner_contact_in_several_companies/models/ir_actions.py:13: [R8110(old-api7-method-defined), IRActionsWindow.read] Method defined with old api version 7
```

**Comment by NL66278** [comment]
No reaction on change requests or questions.

**Pull request closed without merging**